omnibus/config/templates/datadog-agent/upstart_redhat.security.conf.erb

```diff
@@ -15,10 +16,13 @@ script
   #
   # setuid is not available in versions of upstart before 1.4. CentOS/RHEL6 use an earlier version of upstart.
   # This is the best way to set the user in the absence of setuid.
-  exec su -s /bin/sh -c 'DD_LOG_TO_CONSOLE=false exec "$0" "$@"' dd-agent -- <%= install_dir %>/embedded/bin/security-agent start -c <%= etc_dir %>/datadog.yaml -p <%= install_dir %>/run/security-agent.pid &>> /var/log/datadog/security-errors.log
+  exec su -s /bin/sh -c 'DD_LOG_TO_CONSOLE=false exec "$0" "$@"' dd-agent -- <%= install_dir %>/embedded/bin/security-agent start -c <%= etc_dir %>/datadog.yaml -c <%= etc_dir %>/security-agent.yaml -p <%= install_dir %>/run/security-agent.pid &>> /var/log/datadog/security-errors.log
```

Comment:
not a change of this PR, but since I think you want to run the `security-agent` as root this logic to run it as `dd-agent` shouldn't be used. See how it's done for the system-probe: https://github.com/DataDog/datadog-agent/blob/da5454e592bf36e53cc12ec5efd60b31378d92fb/omnibus/config/templates/datadog-agent/upstart_redhat.sysprobe.conf.erb#L35